[Keyboard] Add Keycapsss O4L 5x12 keyboard (#5859)
qmk/qmk_firmware: keyboards/keycapsss/o4l_5x12/readme.md

O4L 5x12
====

> A ortholinear 5x12 keyboard made and sold by Keycapsss. [More info at Keebio](https://keycapsss.com).  

Reviewer: ```suggestion
A ortholinear 5x12 keyboard made and sold by Keycapsss. [More info at Keycapsss.com](https://keycapsss.com).
```

Keyboard Maintainer: BenRoe [Github](https://github.com/BenRoe) / [Twitter](https://twitter.com/ben_roe)  

> Hardware Supported: Pro Micro

Reviewer: Need two spaces at the end of this line for GitHub to render the file properly.

```suggestion
Hardware Supported: Pro Micro  
```

Hardware Availability: [Keycapsss.com](https://keycapsss.com)  

Make firmware .hex for this keyboard (after setting up your build environment):

>     make keycapsss/o4l_5x12/default

Reviewer: `:` is used to separate the keyboard name from the keymap name, not `/`.

```suggestion
    make keycapsss/o4l_5x12:default
```

Example of flashing this keyboard (or use [QMK Toolbox](https://github.com/qmk/qmk_toolbox)):

>     make keycapsss/o4l_5x12/default:avrdude

Reviewer: ```suggestion
    make keycapsss/o4l_5x12:default:avrdude
```

> See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.

Reviewer: Could you update this line, please?

```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
